draft-ietf-quic-qpack.md

```diff
@@ -1261,7 +1272,8 @@ Substantial editorial reorganization; no technical changes.
 - Largest Reference encoded modulo MaxEntries (#1763)
 - New Static Table (#1355)
 - Table Size Update with Insert Count=0 is a connection error (#1762)
-- Stream Cancellations are optional when SETTINGS_HEADER_TABLE_SIZE=0 (#1761)
+- Stream Cancellations are optional when
+  SETTINGS_MAXIMUM_DYNAMIC_TABLE_CAPACITY=0 (#1761)
```

Comment:
Rather than retconning the changelog, it's probably more appropriate to add a current changelog entry that SETTINGS_HEADER_TABLE_SIZE was renamed.